src/System.Net.WebHeaderCollection/tests/WebHeaderCollectionTest.cs

```diff
@@ -283,5 +293,98 @@ public void IterateIEnumerable_Success()
             Assert.Equal("AcceptContent-Length", result);
         }
 
+        [Fact]
+        public void Get_Success()
+        {
+            string[] keys = { "Accept", "uPgRaDe", "Custom" };
+            string[] values = { "text/plain, text/html", " HTTP/2.0 , SHTTP/1.3,  , RTA/x11 ", "\"xyzzy\", \"r2d2xxxx\", \"c3piozzzz\"" };
+            WebHeaderCollection w = new WebHeaderCollection();
+
+            for (int i = 0; i < keys.Length; ++i)
+            {
+                w.Add(keys[i], values[i]);
+            }
+
+            for (int i = 0; i < keys.Length; ++i)
+            {
+                string expected = values[i].Trim();
+                Assert.Equal(expected, w.Get(i));
+            }
+        }
+
+        [Fact]
+        public void Get_Fail()
+        {
+            WebHeaderCollection w = new WebHeaderCollection();
+            w.Add("Accept", "text/plain");
+
+            Assert.Throws<ArgumentOutOfRangeException>(() => w.Get(-1));
+            Assert.Throws<ArgumentOutOfRangeException>(() => w.Get(42));
+        }
+
+        [Fact]
+        public void GetValues_Int_Success()
+        {
+            string[] keys = { "Accept", "uPgRaDe", "Custom" };
+            string[] values = { "text/plain, text/html", " HTTP/2.0 , SHTTP/1.3,  , RTA/x11 ", "\"xyzzy\", \"r2d2xxxx\", \"c3piozzzz\"" };
+            WebHeaderCollection w = new WebHeaderCollection();
+
+            for (int i = 0; i < keys.Length; ++i)
+            {
+                w.Add(keys[i], values[i]);
+            }
+
+            for (int i = 0; i < keys.Length; ++i)
+            {
+                string[] expected = new[] { values[i].Trim() };
+                Assert.Equal(expected, w.GetValues(i));
+            }
+        }
+
+        [Fact]
+        public void GetValues_Int_Fail()
+        {
+            WebHeaderCollection w = new WebHeaderCollection();
+            w.Add("Accept", "text/plain");
+
+            Assert.Throws<ArgumentOutOfRangeException>(() => w.GetValues(-1));
+            Assert.Throws<ArgumentOutOfRangeException>(() => w.GetValues(42));
+        }
+
+        [Fact]
+        public void GetKey_Success()
+        {
+            const string key = "Accept";
+            const string key2 = "Content-Length";
+            WebHeaderCollection w = new WebHeaderCollection();
+
+            w.Add(key, "text/plain");
+            w.Add(key2, "123");
+
+            Assert.Equal(key, w.GetKey(0));
+            Assert.Equal(key2, w.GetKey(1));
+        }
+
+        [Fact]
+        public void GetKey_Fail()
+        {
+            WebHeaderCollection w = new WebHeaderCollection();
+            w.Add("Accept", "text/plain");
+
+            Assert.Throws<ArgumentOutOfRangeException>(() => w.GetKey(-1));
+            Assert.Throws<ArgumentOutOfRangeException>(() => w.GetKey(42));
+        }
+
+        [Fact]
+        public void Clear_Success()
+        {
+            var w = new WebHeaderCollection();
```

Comment:
@Maxwe11 In the rest of the new code, you have this 'WebHeaderCollection w = new WebHeaderCollection();` but here you use a `var`.  So, please be consistent.  You can use the `var` on all the cctor's or none of them.